django/applications/catmaid/models.py

```diff
@@ -391,6 +390,7 @@ class InterpolatableSection(models.Model):
     kept in the stack. The client can the for instance interpolate skeletons in
     the 3D Viewer for this particular section.
     """
+    # TODO: Fix spelling of this class
```

Comment:
I might try to do this before this diff is ready for merge

Reply:
note-to-self: "can the for"

Reply:
Is the issue with "interpolatable"? Wiktionary seems to think it's a valid alternative for "interpolable"; they're both weird words but the current one is a little more clear (it's also used in the java stdlib). Because this spelling is repeated in the table name, changing it would require a migration, which I would personally prefer to avoid if it's just a preference among valid alternatives.

Reply:
Interesting. None of the .. more standard dictionary sites seem to have it. I'm weirded out that Java grabbed onto it. Still, as this has made it into the database, there's little point in changing it now. I'll revise this to a "trail note" for future wanderers in the code.